deployment/terraform/resources/variables.tf

```diff
@@ -53,47 +53,47 @@ variable "k8s_version" {
 # -- Postgres
 
 variable "pg_host" {
-  type = string
```

Comment:
All changes in this file are just whitespace (on one iteration of development, I had added new variables, but then was able to remove them).